### PR TITLE
Some tiny typo fixes and improvements

### PR DESCRIPTION
**Pull request by raphaelbauer**

### CODE REVIEW

File: pagination/Pagination.md [hunk 1]
```diff
@@ -20,15 +20,14 @@ cursor-based over offset-based pagination.
...
 high-data volumes and / or storage in NoSQL databases.
 
-Before choosing offset-based pagination,  make sure that it is feasible for efficient realization.
-Carefully consider the following trade-offs:
+Before choosing cursor-based pagination, consider the following trade-offs:
 
 * Usability/framework support:
 
-    * offset/limit pagination is more familiar than cursor-based, so it has more framework support and
+    * Offset-based pagination is more known than cursor-based pagination, so it has more framework support and
```
Comment (mostlymagic) [inline pagination/Pagination.md]: I'd actually leave in the "limit" to make the point more explicit that we are recommending both of these parameters to be used together

**Comment by mostlymagic** [comment]
LGTM

File: pagination/Pagination.md (hunk 1 above)
Comment (raphaelbauer) [inline pagination/Pagination.md]: Changed it back...

**Comment by raphaelbauer** [comment]
Thanks for the feedback! By the way - the guideline document is very useful and a nice read!

**Comment by whiskeysierra** [comment]
:+1:

**Comment by fmueller** [comment]
Big thank you for your contribution. :)

**Comment by fmueller** [comment]
👍

**Pull request merged**